Source/player.cpp

```diff
@@ -250,8 +250,8 @@ void StartSpell(Player &player, Direction d, WorldTileCoord cx, WorldTileCoord c
 		return;
 	}
 
-	// Checks conditions for spell again, cause initial check was done when spell was queued and the parameters could be changed meanwhile
-	bool isValid = true;
+	// Revalidate the spell conditions

```

Comment:
```suggestion
	// Revalidate the spell conditions, cause initial check was done when spell was queued and the parameters could be changed meanwhile
```

I think the original comment was more helpful because it described why the code was executed. Otherwise you could try to optimize it away completely. 😁